platforms/gke/base/core/container_cluster/container_cluster.tf

```diff
@@ -200,33 +171,13 @@ resource "google_container_cluster" "cluster" {
 
   node_pool {
     initial_node_count = 1
-    name               = "system"
-
-    autoscaling {
-      location_policy      = "BALANCED"
-      total_max_node_count = 32
-      total_min_node_count = 1
-    }
-
-    network_config {
-      enable_private_nodes = true
-    }
-
+    name               = "default-pool"
     node_config {
-      machine_type    = "e2-standard-4"
+      machine_type    = var.cluster_system_node_pool_machine_type
       service_account = google_service_account.cluster.email
```

Comment:
Related to this: can you move `local.cluster_sa_roles` from `platforms/gke/base/core/container_cluster/service_account.tf` to `platforms/gke/base/_shared_config/cluster_variables.tf` so we can reuse that list across services?

In the federated learning use case, we need node pool-specific service accounts, and it would be useful to reuse this list of roles as a baseline. Thanks